lib/components/molecules/Quote/Quote.style.js

```diff
@@ -0,0 +1,22 @@
+import { css } from 'styled-components';
+
+const styles = css`
+  .quote {
+    &__mark {
+      font-size: 60px;
+      float: left;
+    }
+    &__content {
+      padding-top: 5px;
```

Comment:
this should be driven from variables and should be in multiples of a standard multiplier